Modules/_io/bufferedio.c

```diff
@@ -1284,7 +1283,7 @@ _buffered_readline(buffered *self, Py_ssize_t limit)
         Py_CLEAR(res);
         goto end;
     }
-    Py_XSETREF(res, _PyBytes_Join((PyObject *)&_Py_SINGLETON(bytes_empty), chunks));
+    Py_XSETREF(res, PyBytes_Join((PyObject *)&_Py_SINGLETON(bytes_empty), chunks));
```

Comment:
You can pass NULL instead of an empty bytes object. But this should be discussed first.